app/SearchApi/SearchAdapter.Sample/Startup.cs

```diff
@@ -169,12 +169,12 @@ private void ConfigureServiceBus(IServiceCollection services)
             services.AddMassTransit(x =>
```

Comment:
should you use `AddProvider` here ?

Reply:
that's the next ticket.. i am separating them